substrate/runtime/staking/src/lib.rs

```diff
@@ -95,39 +88,26 @@ pub enum LockStatus<BlockNumber: PartialEq + Clone> {
 	Staked,
 }
 
-pub trait ContractAddressFor<AccountId: Sized> {
-	fn contract_address_for(code: &[u8], origin: &AccountId) -> AccountId;
-}
-
-#[cfg(feature = "std")]
-pub struct DummyContractAddressFor;
-#[cfg(feature = "std")]
-impl ContractAddressFor<u64> for DummyContractAddressFor {
-	fn contract_address_for(_code: &[u8], origin: &u64) -> u64 {
-		origin + 1
-	}
+/// The given account was killed.
+pub trait OnAccountKill<Address> {
+	/// The given account was killed.
+	fn on_account_kill(address: &Address);
```

Comment:
should be `AccountId`, not address.

Reply:
Ahh - I see it's a type param... I'd still call it either `A` or `AccountId`

Reply:
Will change that to `AccountId`